### PR TITLE
RELATED: ONE-3410 Fix heatmap hover color from custom palette

### PR DESCRIPTION
**Pull request by ivanmjartan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1893/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1381/ : SUCCESS in 2m 01s

**Comment by ivanmjartan** [comment]
extended test

**Comment by yenkins** [comment]
Demo is running at https://client-demo.na.intgdc.com:50031/storybook/

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1898/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/414/ : FAILURE in 8m 25s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 1]
```diff
@@ -746,6 +745,21 @@ function barSeriesMapFn(seriesOrig: any) {
     return series;
 }
 
+function getHeatMapHoverColor(config: any) {
+    const dataClasses = get(config, ['colorAxis', 'dataClasses'], null);
+    let resultColor = 'rgb(210,210,210)';
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: what about a constant put somewhere near to the HEATMAP_BLUE_COLOR_PALETTE constant?

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 2]
```diff
@@ -746,6 +745,21 @@ function barSeriesMapFn(seriesOrig: any) {
     return series;
 }
 
+function getHeatMapHoverColor(config: any) {
+    const dataClasses = get(config, ['colorAxis', 'dataClasses'], null);
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: return the [] as default and you can remove topmost IF statement on next lines

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/444/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1395/ : SUCCESS in 2m 07s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/782/ : SUCCESS in 15m 25s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/371/ : FAILURE in 20m 46s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/446/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1399/ : SUCCESS in 1m 49s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/785/ : SUCCESS in 14m 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/373/ : SUCCESS in 10m 08s

**Pull request merged**